### ISSUE TITLE
libSavitar fails because DESTDIR environmental var is not set

### ISSUE THREAD

**Issue by jellespijker** [Ultimaker/cura-build-environment#81]
Installing `libSavitor` fails on Arch based distro, because the environmental variable `DESTDIR` is used in `cmake_install.cmake`, which is not set.

**Comment by Ghostkeeper** [comment]
Interesting. I think this is maybe the same reason as why libSavitar and libArcus want to install to `/usr/lib` regardless of what the install path is. I haven't been able to find out what is causing that. There is nothing really special about our CMake in that regard. The mentioned `cmake_install.cmake` is not in our sources.

**Comment by jellespijker** [comment]
The `cmake_install.cmake` is generated by the install instructions provide in `CMakeLists.txt` (see code below) of libSavitar. So maybe it is a libSavitar bug? But the code exempt seems okay by me. 

```cmake
install(TARGETS Savitar
    EXPORT Savitar-targets
    RUNTIME DESTINATION ${CMAKE_INSTALL_BINDIR}
    LIBRARY DESTINATION ${CMAKE_INSTALL_LIBDIR}
    ARCHIVE DESTINATION ${CMAKE_INSTALL_LIBDIR}
    PUBLIC_HEADER DESTINATION ${CMAKE_INSTALL_INCLUDEDIR}/Savitar
)

install(EXPORT Savitar-targets
    DESTINATION ${CMAKE_INSTALL_LIBDIR}/cmake/Savitar
)

```

**Comment by Ghostkeeper** [comment]
We're not calling `CPACK_SET_DESTDIR` as far as I can see, so it shouldn't be using DESTDIR at all :confused:

**Comment by jellespijker** [comment]
@Ghostkeeper  Could you reproduce the issue? Because I can't anymore. I think it happend on my site because I might have forgotten to set the `PKG_CONFIG_PATH`

**Comment by Ghostkeeper** [comment]
I don't think I have the same problem as you. However I still have the problem that if I do `make install` it asks me for admin permissions to create a symlink in `/usr/lib` even though I've changed the `CMAKE_INSTALL_PREFIX` to `~/inst`, which I think is related to your issue.

**Comment by jellespijker** [comment]
Yeah that happend with me as well and I agree that that is unrelated. Could it be that using the  `include(GNUInstallDirs)` (https://cmake.org/cmake/help/v3.17/module/GNUInstallDirs.html) sets the `CMAKE_INSTALL_PREFIX` again to the system paths
